pkg/backingstore/backingstore.go

```diff
@@ -920,6 +905,25 @@ func RunStatus(cmd *cobra.Command, args []string) {
 	}
 }
 
+// GetBackingStoreFromArgs returns the backingstore from CLI arg
+func GetBackingStoreFromArgs(cmd *cobra.Command, args []string) *nbv1.BackingStore {
+	log := util.Logger()
+
+	if len(args) != 1 || args[0] == "" {
+		log.Fatalf(`❌ Missing expected arguments: <backing-store-name> %s`, cmd.UsageString())
+	}
+
+	backStore := util.KubeObject(bundle.File_deploy_crds_noobaa_io_v1alpha1_backingstore_cr_yaml).(*nbv1.BackingStore)
+	backStore.Name = args[0]
+	backStore.Namespace = options.Namespace
+	backStore.Spec = nbv1.BackingStoreSpec{}
+	if !util.KubeCheck(backStore) {
+		log.Fatalf(`❌ Could not get BackingStore %q in namespace %q`,
+			backStore.Name, backStore.Namespace)
+	}
+	return backStore
```

Comment:
Maybe I missed it, but I don't see any check that the backingstore type can be analyzed (pv-pool and STS case)